Makefile

```diff
@@ -78,7 +78,7 @@ run-test-image-pr: # Suppress output as this contains sensitive information
 	-e TRAVIS_BUILD_WEB_URL=$(TRAVIS_BUILD_WEB_URL) \
 	-e TRAVIS_REPO_SLUG=$(TRAVIS_REPO_SLUG) \
 	-e TRAVIS_PULL_REQUEST=$(TRAVIS_PULL_REQUEST) \
-	-e CYPRESS_RBAC_TEST=$(CYPRESS_RBAC_TEST) \
+	-e CYPRESS_RBAC_TEST=true \
```

Comment:
I changed it to true to trigger the tests in e2e. We should bring it back to $(CYPRESS_RBAC_TEST)